README.md

```diff
@@ -1,6 +1,7 @@
-# SEOTools - SEO Tools for Laravel and Lumen
+SEOTools - SEO Tools for Laravel and Lumen
+==========================================
```

Comment:
Why change H1 heading definition style?

Reply:
This does not change document HTML representation, but significantly improves plain-text representation making source '.md' more readable.

Reply:
It only adds inconsistency with h3 headings, IMO.

Reply:
I prefer better source code comprehension to such kind of consistency.
But you are free to switch definition back if you like.
Just don't expect my help for it - I have wasted enough time on this already.